gammapy/makers/safe.py

```diff
@@ -134,6 +139,10 @@ def make_mask_energy_aeff_max(self, dataset):
         """
         geom = dataset._geom
 
+        if self.fixed_offset:
+            self.position = geom.center_skydir.directional_offset_by(position_angle=0.*u.deg,
```

Comment:
Don't set `self.position` here, but just define `position = ...`. I think we should in fact use the offset from the pointing post and not the the offset from the map center...